### ISSUE TITLE
New `Cubist` version available on pip

### ISSUE THREAD

**Issue by Paulnkk** [pjaselin/Cubist#156]
Hey, 

thanks a lot for the amazing and ongoing work on this library. I wanted to ask when a new Cubist version will be available on pip? I want to utilize the nice changes you made concerning https://github.com/pjaselin/Cubist/pull/150 and on branch https://github.com/pjaselin/Cubist/tree/feature/viz ? Since I am currently working with the version of branch https://github.com/pjaselin/Cubist/tree/feature/viz I am not sure how to incorporate the changes from https://github.com/pjaselin/Cubist/pull/150 into this version

Best

Paul

**Comment by pjaselin** [comment]
Hi @Paulnkk I have no schedule for releasing the visualization features as I'm currently working on that. You can take a look at the main branch and see if that suits your needs. It's an unrelease 0.2.0 version and some of the attribute names will change when I merge the viz branch in so that will be tagged as v1.0.0 as it will introduce backwards incompatible changes.

**Comment by Paulnkk** [comment]
Thanks a lot for the response @pjaselin ! If you could release a branch version where the changes you already made concerning the viz extension and 1 rule 1 committe adjustment, I would be very grateful! I am currently writing a paper and it would be great to have this changes included into the numerical experiments.

**Comment by pjaselin** [comment]
Can you check the main branch?On Nov 17, 2024, at 10:30 AM, Paul ***@***.***> wrote:﻿
Thanks a lot for the response @pjaselin ! If you could release a branch version where the changes you already made concerning the viz extension and 1 rule 1 committe adjustment, I would be very grateful! I am currently writing a paper and it would be great to have this changes included into the numerical experiments.

**Comment by Paulnkk** [comment]
I am pulling from this branch: `github.com/pjaselin/Cubist.git@feature/viz` and cubist version is `1.0.0` (version on pip in `0.2.0`), is the latest version available on the main branch of the repo ? 
I do not think that on the latest main branch that the viz changes are included

**Comment by pjaselin** [comment]
Correct, not sure how familiar you are with git. My main branch is unreleased 0.2.0 that I’ve been planning on not releasing unless it meets your needs and the viz branch will be 1.0.0 when released. On Nov 17, 2024, at 5:30 PM, Paul ***@***.***> wrote:﻿
I am pulling from this branch: ***@***.***/viz and cubist version is 1.0.0 (version on pip in 0.2.0), is the latest version available on the main branch of the repo ?
I do not think that on the latest main branch that the viz changes are included
